Makefile

```diff
@@ -14,8 +14,8 @@ clean:
 	rm -rf $(NAME)-$(VERSION)-$(RELEASE).el7.src.rpm
 
 dist:
-	wget https://grafana.com/api/plugins/vonage-status-panel/versions/1.0.5/download \
-	  --output-document=vonage-status-panel-1.0.5-0-g4ecb061.zip
+	wget https://grafana.com/api/plugins/vonage-status-panel/versions/1.0.7/download \
+	  --output-document=vonage-status-panel-1.0.7-0-gd5173e4.zip
```

Comment:
So every time there is an update to vonage plugin, we will have to send such PR?

Cant you simply use the latest instead of specifying the version

Reply:
@TimothyAsir can you suggest a way to do, what @r0h4n is asking for?

Reply:
latest=$( curl -s "https://grafana.com/api/plugins/vonage-status-panel" |
  ruby -rjson -e 'data = JSON.parse(STDIN.read); puts data["links"][-1]["href"]')

Which will always return latest package link like (echo $latest):
/plugins/vonage-status-panel/versions/1.0.7/download
So you can add "https://grafana.com/api" in front of the variable and download

or you can use 
[tim@dhcp35-209 TENDRL]$ latestVersion=$( curl -s "https://grafana.com/api/plugins/vonage-status-panel" |
  ruby -rjson -e 'data = JSON.parse(STDIN.read); puts data["version"]')
[tim@dhcp35-209 TENDRL]$ echo $latestVersion
1.0.7